wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

```diff
@@ -0,0 +1,83 @@
+- Commit your work when you've made reasonably sized changes. It's better to commit an article as a whole rather than 10 small edits.
+- **Use short and meaningful commit messages**, as they let others know what's in the box. Something like `Rewrite the section about jump patterns` says a lot more than `Update en.md`.
+
+## Opening a pull request
+
+A pull request shows other people how your edits will affect the files. Add some information to your pull request to explain your intentions:
+
+- `Title`: a very short descriptive title for your changes in English, together with the article's name. In case of a translation, start with the two-letter language name of your translations in brackets. Examples:
+  - ``[FR] Add `BBCode` ``
+  - ``Update `Beatmapping` and `Beatmap/Difficulty` ``
+- `Description`: anything you want to signal to the maintainers and other potential reviewers. Examples:
+  - A short summary of the changes, especially if they affect several articles
+  - The pull request's completeness, or ideas related to it
+  - [Automatic resolution of relevant issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+- Make sure to tick the `Allow edits from maintainers` checkbox—it will allow the wiki maintainers to help you improve the pull request when necessary
+
+## Resolving conflicts
+
+There are two reasons for why a conflict could have happened:
+
+- You edited a file when your branch was out of date.
+- There was poor communication between you and another person, so you two were editing the same article but that person's changes were merged before yours (causing your edited files to become out of date).
+
+Depending on the severity of the conflicts, you have two options on how to fix this:
+
+1. If your pull request has the `Resolve conflicts` button, click on that. This will open a slightly different version of the web editor.
+   1. GitHub will highlight the conflicting areas. Find one of them.
+   2. Everything from `<<<<<<<` to the `=======` is your changes, whereas everything from `=======` to `>>>>>>> master` is what's in the `ppy/master` branch.
+   3. From here, you will need to manually fix the conflict and delete the lines with the `<<<<<<<`, `=======`, and `>>>>>>> master` markings.
+   4. Repeat the process for all conflicts.
+   5. When completed, click `Mark as resolved` (this is only available when all conflicting parts of the file are resolved).
+2. If the `Resolve conflicts` button is blocked due to the conflicts being too complicated for GitHub, you are out of luck and will need to [update your branch](#syncing-the-fork) and make your changes again.
+   - *Note: This is only true if you are limited to using the GitHub web interface.* There are still ways to fix it, but they don't belong to the scope of this article. Moreover, it is probably not worth the effort to do so, because you will overwrite and revert the conflicting changes.
```

Comment:
```suggestion
   - *Note: This is only true if you are limited to using the GitHub web interface.* There are still ways to fix it, but they don't belong to the scope of this article. Moreover, it is probably not worth the effort to do so, because you will overwrite and revert the conflicting changes.
```
```suggestion
   - *Note: This is only true if you are limited to using the GitHub web interface.* There are still ways to fix it, but they don't belong to the scope of this article. Moreover, it is probably not worth the effort to do so, because you will overwrite and revert the conflicting changes.
   
```
seems like this was missing an ending newline?